backend/api/Services/IsarService.cs

```diff
@@ -99,6 +101,45 @@ public async Task<IsarMission> StartMission(Robot robot, MissionRun missionRun)
             return isarMission;
         }
 
+        public async Task<IsarMission> ReturnHome(Robot robot)
```

Comment:
Consider returning a MissionDefinition or similar instead of IsarMission. Flotilla should only handle MissionDefinition as often as possible to avoid several definitions of missions to be used

Reply:
Made it not return anything